packages/babel-plugin-transform-es2015-parameters/src/rest.js

```diff
@@ -23,6 +23,17 @@ const restLength = template(`
   ARGUMENTS.length <= OFFSET ? 0 : ARGUMENTS.length - OFFSET
 `);
 
+function referencesRest(path, state) {
+  if (path.node.name === state.name) {
+    const ident = path.scope.getBindingIdentifier(state.name);
+    if (!ident || ident === state.outerBinding) {
+      return true;
+    }
```

Comment:
I thought this was covered by the `return false` below. Do you still want me to add one here?